packages/livekit-server-sdk/src/ConnectorClient.ts

```diff
@@ -0,0 +1,283 @@
+  /** Required - The number of the business that is initiating the call */
+  whatsappPhoneNumberId: string;
+  /** Required - The number of the user that is supposed to receive the call */
+  whatsappToPhoneNumber: string;
+  /** Required - The API key of the business that is initiating the call */
+  whatsappApiKey: string;
+  /** Required - WhatsApp Cloud API version, eg: 23.0, 24.0, etc. */
+  whatsappCloudApiVersion: string;
+  /** Optional - An arbitrary string you can pass in that is useful for tracking and logging purposes */
+  whatsappBizOpaqueCallbackData?: string;
+  /** Optional - What LiveKit room should this participant be connected to */
+  roomName?: string;
+  /** Optional - Agents to dispatch the call to */
+  agents?: RoomAgentDispatch[];
+  /** Optional - Identity of the participant in LiveKit room */
+  participantIdentity?: string;
+  /** Optional - Name of the participant in LiveKit room */
+  participantName?: string;
+  /** Optional - User-defined metadata. Will be attached to a created Participant in the room. */
+  participantMetadata?: string;
+  /** Optional - User-defined attributes. Will be attached to a created Participant in the room. */
+  participantAttributes?: { [key: string]: string };
+  /** Optional - Country where the call terminates as ISO 3166-1 alpha-2 */
+  destinationCountry?: string;
+}
+
+export interface AcceptWhatsAppCallOptions {
+  /** Required - The number of the business that is connecting the call */
+  whatsappPhoneNumberId: string;
+  /** Required - The API key of the business that is connecting the call */
+  whatsappApiKey: string;
+  /** Required - WhatsApp Cloud API version, eg: 23.0, 24.0, etc. */
+  whatsappCloudApiVersion: string;
+  /** Required - Call ID sent by Meta */
+  whatsappCallId: string;
+  /** Optional - An arbitrary string you can pass in that is useful for tracking and logging purposes */
+  whatsappBizOpaqueCallbackData?: string;
+  /** Required - The call accept webhook comes with SDP from Meta */
+  sdp: SessionDescription;
+  /** Optional - What LiveKit room should this participant be connected to */
+  roomName?: string;
+  /** Optional - Agents to dispatch the call to */
+  agents?: RoomAgentDispatch[];
+  /** Optional - Identity of the participant in LiveKit room */
+  participantIdentity?: string;
+  /** Optional - Name of the participant in LiveKit room */
+  participantName?: string;
+  /** Optional - User-defined metadata. Will be attached to a created Participant in the room. */
+  participantMetadata?: string;
+  /** Optional - User-defined attributes. Will be attached to a created Participant in the room. */
+  participantAttributes?: { [key: string]: string };
+  /** Optional - Country where the call terminates as ISO 3166-1 alpha-2 */
+  destinationCountry?: string;
+}
+
+// Twilio types
+export interface ConnectTwilioCallOptions {
+  /** The direction of the call */
+  twilioCallDirection: ConnectTwilioCallRequest_TwilioCallDirection;
```

Comment:
we need to re-export this in `index.ts` to ensure users could do `import type { ConnectTwilioCallRequest_TwilioCallDirection } from "livekit-server-sdk"`

otherwise they'd have to do the awkward `import ... from @livekit/protocol`

Reply:
done!